packages/fxa-settings/src/components/FormVerifyCode/index.tsx

```diff
@@ -67,6 +67,7 @@ const FormVerifyCode = ({
 }: FormVerifyCodeProps) => {
   const [isFocused, setIsFocused] = useState<boolean>(false);
   const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
+  const [isDisabled, setIsDisabled] = useState<boolean>(true);
```

Comment:
The input is already wrapped in a form, and the form already supports the idea of valid / not valid, and whether or not it can be submitted. Using the forms built in functionality seems like it might be better than maintaining yet another flag.

Reply:
Took a stab a using the `formData.isValid` but it didn't play well with the current setup. Mostly about when the isValid gets trigger when the user is typing.